deepmd/pt/model/task/fitting.py

```
@@ -302,6 +307,20 @@ def __init__(
         else:
             self.case_embd = None
 
+        if self.default_fparam is not None:
+            if self.numb_fparam > 0:
+                assert len(self.default_fparam) == self.numb_fparam, (
```

Ruff (0.12.2)

Use of `assert` detected

(S101)